com.musala.database.web.spa.spring.parser/src/main/resources/public/endToEndTesting/ProtractorSpecification.js

```diff
@@ -0,0 +1,130 @@
+
+var urlLocation='http://localhost:8080/';
+var packageName='';
+var viewName='index.html';
+
+var origFn = browser.driver.controlFlow().execute;
+browser.driver.controlFlow().execute = function() {
+	var args = arguments;
+	// queue 50ms wait
+	origFn.call(browser.driver.controlFlow(), function() {
+		return protractor.promise.delayed(50);
+	});
+	return origFn.apply(browser.driver.controlFlow(), args);
+};
+
+describe('angularjs homepage', function() {
+	
+  it('should redirect to this', function() {
+	  
+	  browser.get(urlLocation+packageName+viewName);
+
+	  var serverName=  element(by.model('serverName'));
+	  serverName.clear();
+	  serverName.sendKeys('smth');
+	  
+	  var databaseName=  element(by.model('databaseName'));
+	  databaseName.clear();
+	  databaseName.sendKeys('smth');
+	  
+	  var userName=  element(by.model('userName'));
```

Comment:
Please put an empty space before and after every assignment sign (=). Valid for the whole file.